pkgs/swift2objc/lib/src/parser/parsers/declaration_parsers/parse_variable_declaration.dart

```diff
@@ -92,21 +106,21 @@ bool _parsePropertyHasSetter(Json propertySymbolJson) {
   if (hasExplicitGetter) {
     if (hasExplicitSetter) {
       // has explicit getter and has explicit setter
-      return true;
+      return (true, true);
     } else {
       // has explicit getter and no explicit setter
-      return false;
+      return (true, false);
     }
   } else {
     if (hasExplicitSetter) {
       // has no explicit getter and but has explicit setter
       throw Exception(
-        'Invalid property at ${propertySymbolJson.path}. '
+        'Invalid property${path != null ? ' at $path' : ''}. '
         'Properties can not have a setter without a getter',
       );
     } else {
       // has no explicit getter and no explicit setter
-      return true;
+      return (false, false);
```

Comment:
Doesn't this change the behavior? If there's no explicit getter or setter, then we have an implicit getter and setter. That's why this function used to return true here.

In the parse functions you're only looking at `.setter`, not `.getter`, because the rest of the AST assumes that all properties/variables have a getter, and only have a flag for whether there's a setter. So I think that means the AST you'll end up with if there's no explicit getter or setter is that there's an explicit getter (because there's always a getter), but no setter.

Reply:
Yep, that's what I see in the test failures.